### PR TITLE
(FACT-1534) Facter seg faults on Cisco XR

### PR DESCRIPTION
**Pull request by LuvCurves**

### CODE REVIEW

File: acceptance/setup/aio/pre-suite/022_Remove_LD_PRELOAD.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+platforms = hosts.map{|val| val[:platform]}
+skip_test "No Cisco XR hosts present" unless platforms.any? { |val| /^cisco_ios_xr-/ =~ val }
+test_name 'Cisco XR Switch Pre-suite' do
+  switchs = select_hosts({:platform => ['cisco_ios_xr-6-x86_64']})
```
Comment (kurtwall) [inline acceptance/setup/aio/pre-suite/022_Remove_LD_PRELOAD.rb]: Possibly a nit, but I have no idea what `switchs` is intended to connote.

File: acceptance/setup/aio/pre-suite/022_Remove_LD_PRELOAD.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+  step 'remove LD_PRELOAD setting from switch' do
+    switchs.each do |switch|
+      on(switch, "echo 'unset LD_PRELOAD' >> /etc/profile")
+
```
Comment (kurtwall) [inline acceptance/setup/aio/pre-suite/022_Remove_LD_PRELOAD.rb]: I don't think the extra newline is necessary here.

File: acceptance/setup/aio/pre-suite/022_Remove_LD_PRELOAD.rb (hunk 1 above)
Comment (LuvCurves) [inline acceptance/setup/aio/pre-suite/022_Remove_LD_PRELOAD.rb]: The Cisco is a Switch/Router

**Comment by kurtwall** [comment]
+1

**Pull request merged**